stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterRegister.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+/**
+ * Interface to register header names and their corresponding pretty printers
+ */
+public interface AsyncPrettyPrinterRegister {
+
+  /**
+   * Populates AsyncPrettyPrinterRegistry with header names and their corresponding pretty printers
```

Comment:
This is responsible for registering all {@link AsyncPrettyPrinter} to the provided registry.